### PR TITLE
feat: Add more details screen for nodes

### PR DESCRIPTION
**Pull request by jamesarich**
This commit introduces a new "More Details" screen for nodes, accessible via the node chip's context menu.
image Screenshot_20240801_125524 (1)

### CODE REVIEW

File: app/src/main/java/com/geeksville/mesh/model/UIState.kt [hunk 1]
```diff
@@ -194,10 +195,19 @@ class UIViewModel @Inject constructor(
         nodeDB.getNodes(state.sort, state.filter, state.includeUnknown)
     }.stateIn(
         scope = viewModelScope,
-        started = WhileSubscribed(5_000),
+        started = WhileSubscribed(STOP_TIMEOUT_MILLIS),
         initialValue = emptyList(),
     )
 
+    fun getNodeByNum(nodeNum: Int): StateFlow<NodeInfo?> = nodeList.map { list ->
```
Comment (andrekir) [inline app/src/main/java/com/geeksville/mesh/model/UIState.kt]: here prefer the in-memory node DB that is built specifically for this usecase

```kotlin
fun getNodeByNum(nodeNum: Int): StateFlow<NodeInfo?> = nodeDB.nodeDBbyNum.mapLatest { nodes ->
    nodes[nodeNum]
} (...)
```

**Comment by Robert-0410** [comment]
I'm going to start working on this feature, would it be best to open a pull request to this branch? Or should I just open a new pull request when I have something working?

**Comment by jamesarich** [comment]
> I'm going to start working on this feature, would it be best to open a pull request to this branch? Or should I just open a new pull request when I have something working?

Whichever works, you should be able to add my fork as a git remote and pull in the branch to get started.

**Comment by Robert-0410** [comment]
> > I'm going to start working on this feature, would it be best to open a pull request to this branch? Or should I just open a new pull request when I have something working?
> 
> Whichever works, you should be able to add my fork as a git remote and pull in the branch to get started.

Awesome, got that going already thanks.

**Comment by andrekir** [comment]
implemented in #1198

**Pull request closed without merging**